add_catch(test_function_ref test.cpp)
